app/controllers/bookings_controller.rb

```diff
@@ -1,8 +1,9 @@
-# Controller that implements the logic for the bookings of the articles. Articles are booked 
+# Controller that implements the logic for the bookings of the articles. Articles are booked
 # for a particular time frame and charged by a day rate.
 class BookingsController < ApplicationController
   def index
     @bookings = Booking.all
+    @user = current_user
   end
 
```

Comment:
Does the show method need access to a private 'calculate_price' method, so we can show the cost of the booking